lib/web/scripts/node-join/install.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-# Any changes made to this script must also be made to e/lib/web/scripts/install_node.go
```

Comment:
is this no longer the case?

Reply:
No, `install_node.sh` seems to be moved to OSS version https://github.com/gravitational/teleport.e/commit/12b44a5e1c230be12d9ba860870499dcc2c79671